bangazonapi/views/order.py

```diff
@@ -120,8 +121,12 @@ def update(self, request, pk=None):
         """
         customer = Customer.objects.get(user=request.auth.user)
         order = Order.objects.get(pk=pk, customer=customer)
-        order.payment_type = request.data["payment_type"]
-        order.save()
+
+        payment_type_id = request.data.get("payment_type")
```

Comment:
id maybe consider doing a try/excpet with a specific error message of "payment type not found" instead of a generic 404, but this works and looks clean!